Mathlib/NumberTheory/EllipticDivisibilitySequence.lean

```diff
@@ -42,10 +42,10 @@ Some examples of EDSs include
 ## Implementation notes
 
 The normalised EDS `normEDS b c d n` is defined in terms of the auxiliary sequence
-`preNormEDS (b ^ 4) c d n`, which are equal when `n` is odd, and which differ by a factor of `b`
-when `n` is even. This coincides with the definition in the references since both agree for
-`normEDS b c d 2` and for `normEDS b c d 4`, and the correct factors of `b` are removed in
-`normEDS b c d (2 * (m + 2) + 1)` and in `normEDS b c d (2 * (m + 3))`.
+`preNormEDS (b ^ 4) c d n`, which are equal when `n` is odd, and which differ by a factor of an
+extraneous parameter `b` when `n` is even. This coincides with the definition in the references
```

Comment:
I think the "extraneous parameter" here is `b^4` rather than `b`; I suggest you keep the original "differ by a factor of `b`".

I'm not sure "extraneous" has the intended meaning; how about just "parameter" without an adjective?

Reply:
Even "extra" is better IMO, though you probably then want to say "initial values ... and an extra parameter ...".

Reply:
Sorry, sometimes my English short-circuits.